Eliminar el campo de "Añadir claves de contingencia".

OsirisRoman opened the pull request:
De acuerdo a la descripción de los cambios en el facturador electrónico gratuito, el 05/05/2015 se decidió eliminar las claves de uso complementario (contingencia). Esta característica actualmente ya no se usa.

También se actualizan las direcciones de los web services puesto que actualmente el SRI solo maneja facturación electrónica Offline tanto para el Ambiente de Pruebas como para el Ambiente de Producción.

Finalmente se añaden 2 nuevos módulos que mejoran significativamente la interfaz de odoo haciéndola parecerse a la versión enterprise. Estos módulos son gratuitos y fueron contribuidos por Openworx bajo una licencia LGPL-3. Pueden encontrar y descargar estos módulo en la siguiente dirección https://apps.odoo.com/apps/themes/10.0/backend_theme_v10/

Espero que la comunidad siga haciendo crecer este proyecto, aún falta por actualizar la emisión de documentos electrónicos que se encuentra desactualizada y con mucho trabajo por delante.

OsirisRoman committed:
En este commit se elimina el campo de "Añadir claves de contingencia".
De acuerdo a la descripción de los cambios en el facturador electrónico gratuito, el 05/05/2015 se decidió eliminar las claves de uso complementario (contingencia). Esta característica actualmente ya no se usa.

También se actualizan las direcciones de los web services puesto que actualmente el SRI solo maneja facturación electrónica Offline tanto para el Ambiente de Pruebas como para el Ambiente de Producción.

Finalmente se añaden 2 nuevos módulos que mejoran significativamente la interfaz de odoo haciéndola parecerse a la versión enterprise. Estos módulos son gratuitos y fueron contribuidos por Openworx bajo una licencia LGPL-3. Pueden encontrar y descargar estos módulo en la siguiente dirección https://apps.odoo.com/apps/themes/10.0/backend_theme_v10/

Espero que la comunidad siga haciendo crecer este proyecto, aún falta por actualizar la emisión de documentos electrónicos.

ovnicraft commented:
@OsirisRoman esta bien si este PR solo resuelve lo de llaves de contingencia.

Lo de los themes podemos ver donde los ubicamos ya que quedan fuera del scope de este repo.

OsirisRoman commented:
Si porque se elimina el caso en el que los usuarios intenten facturar bajo un modo de indisponibilidad. Esa funcionalidad debía ser removida puesto que el sri actualmente ya no provee llaves de contingencia lo que podía causar errores o confusión de los usuarios al usar el módulo. Lo que si se debe revisar y que me preocupa ligeramente es la actualización del módulo de facturación electrónica, puesto que si ese modelo de contingencia seguía activo me imagino que los archivos de formato xsd necesarios para la facturación electrónica y que también han venido actulizándose con el tiempo pueden estar desactualizados.

ovnicraft commented:
Ok entonces simplifiquemos este PR y dejemos solo los cambios que son respecto a remover las claves.

OsirisRoman commented:
Ok, no hay problema.

OsirisRoman committed:
De acuerdo a la descripción de los cambios en el facturador electrónico gratuito, el 05/05/2015 se decidió eliminar las claves de uso complementario (contingencia). Esta característica actualmente ya no se usa.

También se actualizan las direcciones de los web services puesto que actualmente el SRI solo maneja facturación electrónica Offline tanto para el Ambiente de Pruebas como para el Ambiente de Producción.

Espero que la comunidad siga haciendo crecer este proyecto, aún falta por actualizar la emisión de documentos electrónicos que se encuentra desactualizada y con mucho trabajo por delante.

OsirisRoman commented:
Listo @ovnicraft el PR está simplificado de acuerdo al scope de este repo.

ovnicraft commented:
LGTM
